build.gradle.kts

```diff
@@ -45,34 +45,36 @@ dependencies {
     implementation("io.github.microutils", "kotlin-logging", "2.0.11")
     implementation("org.slf4j", "slf4j-api", "2.0.0alpha1")
     implementation("ch.qos.logback", "logback-classic", "1.3.0-alpha5")
-    implementation("io.sentry", "sentry", "5.1.2")
-    implementation("io.sentry", "sentry-logback", "5.1.2")
+    implementation(platform("io.sentry:sentry-bom:5.2.4"))
```

Comment:
Sentry 5.3.0 is out, any reason you've decided to use 5.2.4?

Reply:
oops, i guess intellij's dependency search lied to me